### DISCUSSION TITLE
Docker image for arm

### DISCUSSION THREAD

**Discussion by darless** [gollum/gollum#1755] [General]
When #1732  was worked on, the user @embolon mentioned:

> Is it possible to build the new docker image for arm/arm64 arch as well?

My response was:

> Docker ARM support.
> Is it possible to build for additional arch - yes. However, I would say that it should be done separately from this pull request, as it will slow down workflows (building 3 arch instead of 1). I did a test where the amd64 workflow to build and test took slightly less than 2 minutes, vs the arm64 and arm took more than 30 minutes each.
> 
> Additionally, when this is merged in, I could look at updating the functionality to build the additional arch, perhaps during deployment and not the main build process. The last point, with the Dockerfile being up to date, any user can build for the system they plan to run it on.

Let's discuss this further here:
- Do we need both arm and arm64?
- Docker for arm has multiple versions: 
  - inux/arm/v6
  - linux/arm/v7
- When this would be done within the deployment, the tag would be the same. Docker will automatically pull down the proper image based on the requesting arch.

**Comment by DanielVolz** [comment]
A image for Raspberry Pi would be great so one could easily install it via docker-compose!

I only found this discussion because I wanted to install a lightweight wiki server on my Pi but only found projects like wiki.js which are to big for a small personal wiki.

Again personally I would appreciate a version of gollum which is compatible with arm.

**Comment by cooperspencer** [comment]
Hi,
There is an arm64 build since yesterday.

**Reply by dometto** [reply]
thanks to @cooperspencer! :)

**Comment by dometto** [comment]
Feel free to let us know here if there are issues with the `arm` build, but think we can mark this discussion as resolved now.
